source/upgrade.txt

```diff
@@ -27,40 +27,16 @@ Before you upgrade, perform the following actions:
   page for this information.
 - Address any breaking changes between the current version of the driver
   your application is using and your planned upgrade version in the
-  section on :ref:`Breaking Changes <java-breaking-changes>` below.
+  section on :ref:`Breaking Changes <java-breaking-changes>` below. To learn
+  more about the MongoDB server release compatibility changes, see the
+  section on :ref:`<java-server-release-changes>`.
```

Comment:
Issue: Per [style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#term-about), don't use "on" to mean "about."

Suggestion: 
```suggestion
  more about the MongoDB server release compatibility changes, see :ref:`<java-server-release-changes>`.
```

Reply:
Good catch. I'll change it to "java-server-release-changes section". When linking content, I prefer to identify the type so that the reader has some idea of what they're going to be looking at or for (guide, tutorial, page, section, etc.).